piratebartender/js/app.js

```diff
@@ -0,0 +1,160 @@
+        includedItems.push(newPantry.stock[randomIndex])
+        $(this).parent("div").hide();
+        console.log(includedItems);
+    })
+
+
+    $('#Salty').click(function() {
+        $(this).parent("div").hide()
+        var randomIndex = Math.floor(Math.random() * (3 - 1 + (1)));
+        includedItems.push(salty.options[randomIndex]);
+        console.log(includedItems);
+    })
+
+    $('#Spicy').click(function() {
+        $(this).parent("div").hide()
+        var randomIndex = Math.floor(Math.random() * (3 - 1 + (1)));
+        includedItems.push(spicy.options[randomIndex]);
+        console.log(includedItems);
+
+    })
+
+    $('#saltyno').click(function() {
+        var randomIndex = Math.floor(Math.random() * (6 - 1 + (1)));
+        includedItems.push(newPantry.stock[randomIndex])
+        $(this).parent("div").hide();
+        console.log(includedItems);
+    })
+
+    $('#spicyno').click(function() {
+        var randomIndex = Math.floor(Math.random() * (6 - 1 + (1)));
+        includedItems.push(newPantry.stock[randomIndex])
+        $(this).parent("div").hide();
+        console.log(includedItems);
+    })
+
+    $('#Sour').click(function() {
+        $(this).parent("div").hide()
+        var randomIndex = Math.floor(Math.random() * (3 - 1 + (1)));
+        includedItems.push(sour.options[randomIndex]);
+        console.log(includedItems);
+    })
+
+  
+    $('#sourno').click(function() {
+        var randomIndex = Math.floor(Math.random() * (6 - 1 + (1)));
+        includedItems.push(newPantry.stock[randomIndex])
+        $(this).parent("div").hide();
+        console.log(includedItems);
+    })
+
+    //last question, answer will trigger function
+     $('#Strong').click(function() {
+        $(this).parent("div").hide()
+        var randomIndex = Math.floor(Math.random() * (3 - 1 + (1)));
+        includedItems.push(strong.options[randomIndex]);
+        console.log(includedItems);
+        makeDrink();
+    })
+
+    $('#strongno').click(function() {
+        var randomIndex = Math.floor(Math.random() * (6 - 1 + (1)));
+        includedItems.push(newPantry.stock[randomIndex])
+        $(this).parent("div").hide();
+        console.log(includedItems);
+        makeDrink();
+    })
+
+    //drink naming arrays
+    var drinkNouns = ['Witch', 'Bastard', 'Thief', 'Murderess', 'Phantom', 'Raven', 'Cryptkeeper', 'Scream', 'Mystery', 'Demon'];
+    var drinkAdjectives = ['Doomed', 'Gloomy', 'Melancholic', 'Dark', 'Funereal', 'Gruesome', 'Blood-curdling', 'Ghoulish', 'Revolting', 'Sinister']
+
+
+    //function to make the drink, including the name
+    var makeDrink = function(ingredients) {
+        $(".serve").append("<h3> Please, enjoy your special beverage, made with:</h3>")
+        for (i = 0; i < includedItems.length; i++) {
+
+            $(".serve").append("<p>" + includedItems[i] + "</p>")
+
+        }
+
+        var randomIndex = Math.floor(Math.random() * (9 - 1 + (1)));
```

Comment:
You could generate two randomIndex numbers for drinkNouns and a different one for drinkAdjectives and really mix things up!

_I dig it_